packages/commonwealth/server/scripts/purgeCloudflareCache.ts

```diff
@@ -0,0 +1,41 @@
+import { logger } from '@hicommonwealth/core';
+import fetch from 'node-fetch';
+import { config } from '../config';
+
+const log = logger(import.meta);
+
+async function purgeCache(zoneId?: string, apiKey?: string) {
+  if (!zoneId || !apiKey) {
+    log.warn('Missing Cloudflare env variables. Skipping cache purge');
+    return;
```

Comment:
should this be a warning or an error?

Reply:
Do we need cache purging in non-production environments (is there a cache to purge)? If not we should just remove the log entirely since the env var are guaranteed to exist in production at this point.

Reply:
Yeah basically we don't need to purge non-prod environment, but I figured we should probably have some check just in case we expect it to purge on a non-prod environment, because this script will always run but not execute. I think its good for debugging purposes.